pkg/awsutils/awsutils.go

```diff
@@ -36,6 +37,7 @@ import (
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/prometheus"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Can we move this to events.k8s.io v1. Upstream seems to be moving away from corev1/Events to events.k8s.io/Events. ref:https://github.com/kubernetes/kubernetes/pull/102832